### PR TITLE
Added desktop version

### PR DESCRIPTION
**Pull request by ahmadsarfrazx**
- Added a new branch named desktop-version
- Added a new file named query.css
- Add media queries in query.css file to make desktop portfolio responsive also identical to Figma
- Removed the width from the .container element as per the requirement
- Reduced the white space from project description and cards
- Removed the bottom line from the upside of the framework item
- Made all required changes
- Adjusted border radius as per requirement which matches the Figma template
- Implemented the dropdown icons as suggested by the previous reviewer
- All required changes have been implemented successfully and the project is complete now
- Removed list style in the about section
- Changed language icon from column to row

### CODE REVIEW

File: query.css [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+  }
+
+  #order-3 {
+    display: none;
+  }
+
+  /* end card */
+
+  /* about */
+  .about {
```
Comment (bienvenuushindi) [inline query.css]: :red_circle: Kindly notice the top-right border radius on this section. Please make sure you add the radius to match the Figma design

Your Design
border

Figma Design
borderfigma

File: query.css [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+  }
+
+  .about-right-section {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    margin-left: 200px;
+  }
+
+  .about_ul {
```
Comment (bienvenuushindi) [inline query.css]: :red_circle: Kindly notice that each language item has an icon at the left. Your design has no icon for this section as you can in this screenshot

Your Design
yrnoicon

Figma Design
noarrows

:red_circle: Please make sure you include the icons as shown on this Figma design :arrow_up_small:

Comment (bienvenuushindi) [reply query.css]: :red_circle: Kindly notice that the bottom line is only applied to the last two items (Frameworks & Skills). Please make sure you remove the bottom line on the first item (language)

Your Design
yrborderless

Figma Design
figborderless

File: query.css [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+  .connect_button {
+    margin-left: 200px;
+    font-size: 20px;
+  }
+
+  .social ol {
+    margin-left: 200px;
+  }
+
+  .desktop-card,
```
Comment (bienvenuushindi) [inline query.css:98]: :red_circle: Kindly notice the white space after the project description on odd cards and before on even cards. Kindly reduce the white space to match the Figma design

Your Design
yrextraspace

Figma Design

extraspace

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @ahmadsarfrazx ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- [ ] N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: query.css [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+    color: white;
+    font-size: 17px;
+    transition: 100ms;
+    cursor: pointer;
+  }
+
+  .end-line {
+    display: flex;
+  }
+}
```
Comment (bienvenuushindi) [inline query.css:375]: :red_circle: Kindly make sure you remove the fixed width applied on the` .container` element as it makes your design unresponsive when resizing your screen. You can see its effect in the screenshot below :arrow_down: 
chnage

**Comment by ahmadsarfrazx** [comment]
- Removed the width from the .container element as per the requirement
- Reduced the white space from project description and cards
- Removed the bottom line from the upside of the framework item
- Made all required changes

File: query.css (hunk 1 above)
Comment (Debas-31) [reply query.css]: - [ ] Make sure this issue is addressed with the feedback from the previous reviewer about the border radius for the top right corner based on the Figma template design; please fix this issue

File: query.css (hunk 2 above)
Comment (Debas-31) [reply query.css]: - [ ] Please implement icons in the above section, asked to work by the previous reviewer :+1:

**Review by Debas-31 — CHANGES_REQUESTED** [review]

**Review by Debas-31 — CHANGES_REQUESTED** [review]
> ## Hi @ahmadsarfrazx :wave:, 
# Status: Require Change :recycle:
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- Linters setup :heavy_check_mark:
- GitHub flow :heavy_check_mark: 
- Commit message :heavy_check_mark:
 ## Required Changes ♻️
_Check the comments above the review._

## Optional suggestions
- No 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmadsarfrazx** [comment]
- Adjusted border radius as per requirement which matches the Figma template
- Implemented the dropdown icons as suggested by the previous reviewer
- All required changes have been implemented successfully and the project is complete now

File: index.html [hunk 5]
```diff
@@ -163,74 +164,95 @@ <h2 class="title">Multi-Post Stories</h2>
...
+              JavaScript
+            </li>
+            <li>
+              <img src="assets/html.svg" alt="HTML" />
+              HTML
+            </li>
+            <li>
+              <img src="assets/css.svg" alt="CSS" />
+              CSS
+            </li>
```
Comment (Olamarx) [inline index.html:231]: - [ ] While you were implementing the changes, you forgot to remove the list style(dots beside the languages, framework and skills) also, Javascript, html and CSS are in rows, not column.
### Your webpage
image
### Figma
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @ahmadsarfrazx, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- All linters passed ✔️ 
- Good commit messages 👍🏼 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmadsarfrazx** [comment]
- Removed list style in the about section
- Changed language icon from column to row

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @ahmadsarfrazx , 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**